README.rst

```diff
@@ -776,6 +776,31 @@ model, then configure the setting as following:
 This setting allows to disable creating `notification preferences <#notification-preferences>`_
 on running migrations.
 
+``OPENWISP_NOTIFICATIONS_NOTIFICATION_STORM_PREVENTION``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A ``dict`` containing configuration for notification storm prevention mechanism.
+The default configuration is as follows:
+
+.. code-block:: python
+
+    OPENWISP_NOTIFICATIONS_NOTIFICATION_STORM_PREVENTION = {
```

Comment:
We might need to re-write this documentation. I couldn't explain it really well 😓